_posts/2023-11-30-prevent-broker-scale-down-if-containing-paritition-replicas.md

```diff
@@ -0,0 +1,179 @@
+---
+layout: post
+title: "Preventing Kafka broker scale down if partition replicas are present"
+date: 2023-12-11
+author: Shubham Rawat
+---
+
+Apache Kafka is a platform designed for scalability.
+You can always scale in or scale out your Kafka Clusters based on your use case.
+When scaling down a cluster, it's crucial to ensure that data across brokers is moved or copied throughout the cluster.
+You may experience data loss if you forget to move out partition replicas from the broker being removed.
+But don't worry, with Strimzi 0.38, we have introduced the broker scale down check which is going to take care of this problem.
+
+## Broker scale down check
```

Comment:
I wonder if it would be better to keep this for the end - before the "What's next" chapter. I find it a bit weird that you have a blog post introducing a new feature and you start with _how to disable it_. I would first explain the feature and at the end say _if needed, you can disable it_.